.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+TAG=${GITHUB_REF_NAME}
+# The prefix is chosen to match what GitHub generates for source archives
+PREFIX="rules_nixpkgs-${TAG:1}"
+ARCHIVE="rules_nixpkgs-${TAG:1}.tar.gz"
+git archive --format=tar.gz --prefix="${PREFIX}/" -o $ARCHIVE ${TAG}
+SHA=$(shasum -a 256 "$ARCHIVE" | awk '{print $1}')
+
+cat << EOF
```

Comment:
If you single quote the heredoc marker:
```suggestion
cat << 'EOF'
```
You can avoid needing to escape the backticks below.

Reply:
There are some variable expansions needed in the heredoc and `<<'EOF'` will keep those verbatim. E.g.
```
$ FOO=foo
$ cat <<EOF
> foo $FOO
> EOF
foo foo
$ cat <<'EOF'
foo $FOO
EOF
foo $FOO
```